docs/custom_quantum_functions.md

```diff
@@ -471,3 +292,15 @@ If registrations fail, then you can increase this value in your keyboard or keym
 ```c
 #define MAX_DEFERRED_EXECUTORS 16
 ```
+
+# Legacy Content :id=legacy-content
+
+This page used to encompass a large set of features. We have moved many sections that used to be part of this page to their own pages. Everything below this point is simply a redirect so that people following old links on the web find what they're looking for.
```

Comment:
Labeling these links as "Legacy Content" may cause some readers to infer that these functions are deprecated (which is certainly not the case). It would be better to keep the section headers where they are now, but with only a short summary of the functionality, and a link to the new location of the documentation.

Reply:
I used the same wording as in `docs/feature_advanced_keycodes.md`, I'll do the modification you asked

Reply:
I changed the title of the section, WDYT?

Reply:
I think that's better! Thanks!